constants.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  // GraphQL nodes type for entities with type "post"
+  POST_NODE_TYPE: "OnlinesalesPost",
```

Comment:
Onlinesales -> OnlineSales - this is a proper way to write the name of the product

Reply:
Fixed